themes/src/main/resources/theme/base/login/login-password.ftl

```diff
@@ -33,11 +39,12 @@
                     </div>
 
                     <div id="kc-form-buttons" class="${properties.kcFormGroupClass!}">
-                    <input tabindex="4" class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonBlockClass!} ${properties.kcButtonLargeClass!}" name="login" id="kc-login" type="submit" value="${msg("doLogIn")}"/>
-                  </div>
-            </form>
+                        <input tabindex="4" class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonBlockClass!} ${properties.kcButtonLargeClass!}" name="login" id="kc-login" type="submit" value="${msg("doLogIn")}"/>
+                    </div>
+                </form>
+            </div>
         </div>
-      </div>
+        <script type="text/javascript" src="${url.resourcesPath}/js/passwordVisibility.js"></script>
```

Comment:
Ideally this script tag would be included at the very bottom of the document, preferably before the `</body>` tag. This prevents the script from blocking the page loading, as browsers will stop evaluating a document if they encounter a `<script>` tag, and wait until it is downloaded and evaluated.

I notice that this done inconsistently throughout the theme, so I would like to fix this first. Perhaps we can make a function for the template to register enqueue these kind of scripts.

Reply:
I think this can be a separate discussion (see #22042). I think considering the current state of loading scripts in the theme, we can continue in the way this PR works at the moment in regards to the way scripts are included.